client/public/locales/es/translation.json

```diff
@@ -77,23 +77,23 @@
   "Game Rules": "Reglas del juego",
   "levels": "niveles",
   "tactics": "tácticas",
-  "regular": "normal",
-  "relaxed": "relajado",
-  "none": "ninguno",
-  "<p>Select a JSON file with the saved game progress to load your progress.</p><1><0>Warning:</0> This will delete your current game progress! Consider <2>downloading your current progress</2> first!</1>": "<p>Seleccione un archivo JSON con el progreso del juego guardado para cargar su progreso.</p><1><0>Advertencia:</0> Esto borrará su progreso actual en el juego. Considere <2>descargar su progreso actual</2> antes</1>",
+  "regular": "normales",
+  "relaxed": "relajadas",
+  "none": "ninguna",
+  "<p>Select a JSON file with the saved game progress to load your progress.</p><1><0>Warning:</0> This will delete your current game progress! Consider <2>downloading your current progress</2> first!</1>": "<p>Selecciona un archivo JSON con el progreso del juego para cargarlo.</p><1><0>Advertencia:</0> Esto borrará tu progreso actual en el juego. ¡Considera antes <2>descargar tu progreso actual</2>!</1>",
   "Upload Saved Progress": "Subir progreso guardado",
   "Load selected file": "Cargar archivo seleccionado",
   "Rules": "Reglas",
-  "<p>As this server runs lean on our university machines, it has a limited capacity. Our current estimate is about 70 simultaneous games.</p>": "<p>Como este servidor corre en máquinas de nuestra universidad, tiene una capacidad limitada. Nuestra estimación actual es de unos 70 juegos simultaneos.</p>.",
-  "<0>Most aspects of the games and the infrastructure are still in development. Feel free to file a <1>GitHub Issue</1> about any problems you experience!</0>": "<1>Muchos aspectos de los juegos y la infrastructura están aún en desarrollo. No dude en abrir una <1>GitHub Issue</1> sobre cualquier problema que experimente.</1>",
-  "This server has been developed as part of the project <1>ADAM: Anticipating the Digital Age of Mathematics</1> at Heinrich Heine University Düsseldorf.": "Este  servidor se ha desarrollado como parte del proyecto <1>ADAM: Anticipating the Digital Age of Mathematics</1> en la Heinrich-Heine-Universität de Düsseldorf.",
-  "Server capacity": "",
-  "RAM": "",
-  " used": "",
-  "CPU": ""
+  "<p>As this server runs lean on our university machines, it has a limited capacity. Our current estimate is about 70 simultaneous games.</p>": "<p>Como este servidor corre en máquinas de nuestra universidad, tiene una capacidad limitada. Nuestro estimado actual es de unos 70 juegos simultáneos.</p>.",
```

Comment:
"Estimación" is the wrong word, it's related to "esteemed" rather than "estimated". "Simultaneos" has an accent!